### DISCUSSION TITLE
动态相册在新版本部署报错，请求援助

### DISCUSSION THREAD

**Discussion by Oilan94** [valor-x/hexo-theme-solitude#468] [Q&A]
当前主题使用main2.1.5版本，按照博主亦小封“[Solitude魔改教程：iCat同款动态相册页](https://blog.meuicat.cn/posts/648b1ceb.html)”，相册页面能正常显示，但是网站出现一些问题，版主貌似不常看博客，只能来这里请求援助。
1.网站开启懒加载后图片加载失败
2.waterfall.min.js内 `i(t) !== containerWidth && (window.removeEventListener(e.type, f),`部分报错
3.pjax.min.js `el.dispatchEvent(event);` `return fn.call(context, els)` ` forEachEls(els, function(el) {` `events.forEach(function(e) {` ` trigger(window, "resize scroll");` `this.onSwitch()` `this.onSwitch()`部分报错。
4.报错名称：Uncaught ReferenceError: containerWidth is not defined、Uncaught TypeError: Cannot read properties of null (reading 'children')
5.
![报错截图1](https://github.com/user-attachments/assets/af85bb63-2d71-4dae-b70f-bcc14cb555a7)
![报错截图2](https://github.com/user-attachments/assets/771ddaf7-1945-4344-bcda-d3e3449ba91d)

**Comment by yife68** [comment]
这个是魔改的问题，不宜在主题讨论。

**Reply by Oilan94** [reply]
那怎么能方便请教一下，实在是没地方去了，很喜欢这个功能。

**Reply by yife68** [reply]
是在不会修，建议再等等。后续会更新教程。

**Comment by yife68** [comment]
是在不会修，建议再等等。后续会更新教程。

**Reply by Oilan94** [reply]
好的，只要有新教程就有底了，提前感谢了。
